notebooks_tsqr/consdb/assorted_plots.ipynb

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from IPython.display import display, Markdown, display_markdown
</pre><p>remove unused commits, add isort to pre-commit</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>